modules/benchmark/R/calc.metrics.R

```diff
@@ -12,44 +11,74 @@
 ##' 
 ##' @author Betsy Cowdery
 
-calc.metrics <- function(data.path, format, vars_used, model_run, metrics, start_year, end_year, site){
+calc.metrics <- function(data.path, format, model_run, metrics, start_year, end_year, site, bm, ens){
   
-  # Right now many argument are being read in as R list objects
-  # This won't work remotely but at the moment I don't have a good solution
+  # Right now I'm making the inappropriate assumption that storage type will be 
+  # empty unless it's a time variable. 
+  # This is because I haven't come up for a good way to test that a character string is a date format
```

Comment:
As per recent discussion about met2CF.csv with @ankurdesai re #888, I think there's an explicit need for the format record to have an one variable that matches BETY variable datetime and the units on that variable should point to the conversion function for that particular data format. Will need to eventually generalize to the case where datetime info is stored across multiple columns or variables.